pkg/cover/backend/dwarf.go

```diff
@@ -124,6 +126,26 @@ func processModule(params *dwarfParams, module *Module, info *symbolInfo,
 	return result, nil
 }
 
+// GCC < 14 incorrectly tail-calls kcov callbacks, which does not let syzkaller
+// verify that collected coverage points have matching callbacks.
+// See https://github.com/google/syzkaller/issues/4447 for more information.
+func IsKcovBrokenInCompiler(versionStr string) bool {
+	// Some targets (e.g. NetBSD) use g++ instead of gcc.
+	var gccRE = regexp.MustCompile(`gcc|GCC|g\+\+`)
```

Comment:
Nit: It may be better to put these `regexp.MustCompile` calls out of the function.

Reply:
Done.